### PR TITLE
[mergebot cli] add new revert regex

### PR DESCRIPTION
**Pull request by zengk95**
Since we added a new CLI, we have to update the regex validator for reverting. The same validation is not there for merging. 

The example command we want to validate:
`@pytorchbot merge -c="ghfirst" -m="this is breaking things internally"`

### CODE REVIEW

File: .github/scripts/trymerge.py [hunk 1]
```diff
@@ -301,6 +301,7 @@
     re.MULTILINE
 )
 RE_REVERT_CMD = re.compile(r"@pytorch(merge|)bot\s+revert\s+this")
+RE_REVERT_CMD_CLI = re.compile(r"@pytorch(merge|)bot\s+revert\s+(-m.*|-c.*)(-m.*|-c.*)")
```
Comment (atalman) [inline .github/scripts/trymerge.py:304]: Question, why (-m.*|-c.*)(-m.*|-c.*) is included 2 times ?
We are trying to match :
pytorchbot revert -m "Test" -c "test"
pytorchbot revert -c "Test" -c "test"

**Review by atalman — COMMENTED** [review]
Can you please add description to the PR summary of the use case we are trying to address

File: .github/scripts/trymerge.py [hunk 2]
```diff
@@ -870,7 +871,7 @@ def merge_on_green(pr_num: int, repo: GitRepo, dry_run: bool = False) -> None:
             print(f"Merged failed due to: {ex}. Retrying in 60 seconds.")
             time.sleep(60)
     # Finally report timeout back
-    msg = "Merged timed out after 6 hours. Please contact the pytorch_dev_infra team."
+    msg = "Merged timed out after 400 minutes. Please contact the pytorch_dev_infra team."
```
Comment (malfet) [inline .github/scripts/trymerge.py:874]: Nit: can you make a timeout_min default argument of `merge_on_green` and change this one into f-string?
```suggestion
    msg = f"Merged timed out after {timeout_min} minutes. Please contact the pytorch_dev_infra team."
```

**Review by malfet — APPROVED** [review]

**Review by janeyx99 — APPROVED** [review]

File: .github/scripts/trymerge.py (hunk 1 above)
Comment (zengk95) [reply .github/scripts/trymerge.py:304]: In my mind, it makes sense to do it only on the test-infra side but maybe they wanted to have some additional checks here as well? Like they check if the comments been edited as well which can be a useful check.

**Pull request closed without merging**